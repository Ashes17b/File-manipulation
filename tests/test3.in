Input file: empty or not? 
    - error need delete.