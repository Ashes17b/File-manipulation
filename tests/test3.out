    - need delete.
Input file: empty or not? 
